Symptom changes

zeskorion opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
-nerfed sneezing
-small buff to hyphema
## Why It's Good For The Game
-Hyphema is a fun symptom, but impossible to spread to its full potential without sneezing, even though it is possible to kill everyone quit easily without sneezing, oddly enough
-sneezing makes spreading a virus way too easy, especially for a starter symptom
## Changelog
:cl:
balance: Nerfed Sneezing, now requires 9 transmission to spread
balance: Lowered hyphema symptom threshold to 11

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

zeskorion committed:
Small buff

Lowers eye popping threshold from 12 to 11, to make it possible to spread without sneezing

zeskorion committed:
Nerfed Sneezing

sneezing now only spreads a virus with a transmission greater than nine

ike709 commented on code/datums/diseases/advance/symptoms/sneeze.dm:
```diff
@@ -46,7 +55,3 @@ Bonus
 		if(1, 2, 3)
 			if(!suppress_warning)
 				M.emote("sniff")
-		else
```
Just do `if(A.properties["transmittable"] >= 9)` inside this `else` and revert all other changes to this file.

ike709 replied:
Otherwise people will only sneeze when it first starts processing.

ike709 commented:
Also needs bee edit comments.

zeskorion committed:
Sneezing nerf quickfix

should actually work now. sneezing only spreads at trans 9. should not spread at all though, imo

zeskorion committed:
bee edit flags

added edit flags to buff (lowered symptom threshold)

zeskorion committed:
Working as intended

wiki notes that necrotizing fasciitis scales inversely with stealth. the framework existed in the code, but did not work itself. I fixed the framework and it should now work
